txstratum/prometheus.py

```diff
@@ -47,13 +47,13 @@
     "txs_solved_weight": Histogram(
         "txs_solved_weight",
         "Txs solved histogram by tx weight",
-        buckets=(17, 18, 19, 20, 21, 22, 23, 25, float("inf")),
+        buckets=(17, 18, 19, 20, 21, 22, 23, 25, 26, 27, 28, 29, 30, float("inf")),
```

Comment:
Should we add 24 as well? Since apparently you wanted a step of 1

Reply:
You are correct, I didn't notice this gap. Done https://github.com/HathorNetwork/tx-mining-service/pull/115/commits/6dd95496da371543d95e57c529eac07dd3c05d6a